packages/frontend/src/features/scheduler/components/SchedulersListItem.tsx

```diff
@@ -61,12 +61,22 @@ const SchedulersListItem: FC<SchedulersListItemProps> = ({
                     </Group>
                 </Stack>
                 <Group noWrap spacing="xs">
-                    <Switch
-                        checked={scheduler.enabled}
-                        onChange={() => {
-                            mutateSchedulerEnabled(!scheduler.enabled);
-                        }}
-                    />
+                    <Tooltip
+                        label={
+                            scheduler.enabled
+                                ? 'Toggle off to temporarily pause notifications'
+                                : 'Notifications paused. Toggle on to resume'
+                        }
+                    >
+                        <div>
```

Comment:
required div for the toggle to work... is there a better way ?

Reply:
You can/should use Box, but div is ok. Mantine wants a non-disabled ref.

Reply:
thanks for the tip. I updated the code